compiler/quilt/tools/command.py

```diff
@@ -68,7 +69,18 @@
 
 LOG_TIMEOUT = 3  # 3 seconds
 
-VERSION = pkg_resources.require('quilt')[0].version
+try:
+    VERSION = pkg_resources.require('quilt')[0].version
+except pkg_resources.DistributionNotFound:
+    # Not installed, so we'll get the version manually.
+    try:
+        filedir = os.path.split(os.path.abspath(__file__))[0]
+        quiltdir = os.path.split(filedir)[0]
+        pkgdir = os.path.split(quiltdir)[0]
+        VERSION = open(os.path.join(pkgdir, 'VERSION')).read().strip()
```

Comment:
This looks similar to the helper function below?

Reply:
* they both operate when not installed
* `quilt.tools.command` can't reasonably be imported from `setup.py`
* `setup.py` can't be reasonably imported from `command.py`

...can you be more specific with what you'd like me to do here?